google-cloud-examples/src/main/java/com/google/cloud/examples/compute/README.md

```diff
@@ -0,0 +1,3 @@
+[**DEPRECATED**]
+
+The GAPIC compute samples have been moved to [Java-docs-samples repository](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/compute/cloud-client).
```

Comment:
This is a good note. But "GAPIC" is not good term for customers.

Reply:
Thanks. Modified it.